### ISSUE TITLE
About HyperParams

### ISSUE THREAD

**Issue by ftgreat** [RUC-GSAI/YuLan-Mini#10]
![Image](https://github.com/user-attachments/assets/1022ee57-5d39-41d1-bd0c-3ddbffa7007f)

Hello, thank you for your paper. As shown in the figure, the paper references the hyperparameter settings of SI. The original SI paper mentions methods such as Scaled Embedding, but it seems that the version of SI in your paper is the most basic one. Is my understanding correct? Thank you.

**Comment by huyiwen** [comment]
> the version of SI in your paper is the most basic one.

Yes.

Scaled initialization is a simple concept where we scale down matrices that contribute to training instability. Similar approaches can be found in many existing papers.

> The original SI paper mentions methods such as Scaled Embedding

While the original scaled embedding **scales up** initialization values, we empirically found that **scaling down** the embedding matrix initialization (e.g. 1e-5) may improve convergence.

Btw, another confusing point in the original paper is embedding layernorm, which creates two consecutive layer normalization operations in pre-norm architectures.

**Comment by huyiwen** [comment]
Oh my bad, but I am still uncertain about its effectiveness in terms of training stability.
